HiP-App/HiP-Forms
# fixing HIPM-1006: Notification: no network connnection

Fixed bug HIPM-1006.

**Test cases:**
- Start the HiP-App.
- If user was logged in, only server error message should have been shown (logged in users currently get a "(500) Internal Server Error" from server on API call achievements/api/Achievements/Unlocked)
- Logout out via Menu | Account | "LOGOUT" | "Do you really want to logout?": "YES". No error message should be shown after logging out.
- Restart App. Again no error message should be shown during App start.
- Login in again via Menu | Account | "LOG IN WITH DEBUG LOGIN". Again only the server error message should be shown now.

**What was done in detail:**
* Made error dialog usage to be more precise: "(500) Internal Server Error" is not a problem with internet, and also not a network connection problem, which the error message content Alert_No_Internet was used for, but a server problem, for which Alert_Server_Error perfectly matched.
* Localised all 3 sources of the triggered error message (and roughly fixed the cases for a not logged in user): During AppStart the Achievements are trying to be fetched (incl. Achievements/Unlocked call lead to a http status 500), no matter if a user is currently logged in or not, and also after logging out during runtime.
* Talked to team Fox about the "500 InternalServerError", from Achievements/Unlocked in the case when there are no achievements unlocked for response, yet. When this is fixed, no HttpStatusCode.InternalServerError exception will be trown any longer by AchievementsScreenViewModel and the currently shown server error message will also no be shown any longer.
